tools/docker/image-build.sh

```diff
@@ -19,12 +19,11 @@ usage() {
     echo "      -h|--help - show this help menu"
     echo "      -v|--verbose - verbosity on"
     echo "      -b|--base-image - Base OS image to use (Dockerfile 'FROM')"
-    echo "      -n|--native - Use the same base OS image as the running system"
```

Comment:
As mentioned in the other PR: Please include an explanation in the commit message why this didn't work.